MovieApp/LogInViewController.swift

```diff
@@ -0,0 +1,143 @@
+//
+//  LogInViewController.swift
+//  MovieApp
+//
+//  Created by endava-bootcamp on 28.03.2023..
+//
+
+import Foundation
+import UIKit
+import PureLayout
+import MovieAppData
+
+class LogInViewController: UIViewController {
+    
+    private var mainLabel: UILabel!
+    private var emailLabel: UILabel!
+    private var passLabel: UILabel!
+    private var emailField: UITextField!
+    private var passField: UITextField!
+    private var button: UIButton!
+    
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        buildView()
+    }
+    
+    func signInLabel(){
+        mainLabel = UILabel()
+        view.addSubview(mainLabel)
+        mainLabel.autoAlignAxis(toSuperviewAxis: .vertical)
+        mainLabel.textColor = .white
+        mainLabel.font = UIFont.systemFont(ofSize: 24)
+        mainLabel.textAlignment = .center
+        mainLabel.text = "Sign in"
+        
+        mainLabel.autoPinEdge(toSuperviewSafeArea: .top, withInset: 50)
+        mainLabel.autoPinEdge(toSuperviewSafeArea: .leading, withInset: 12)
+    }
```

Comment:
```suggestion
    func createViews(){
        mainLabel = UILabel()
        view.addSubview(mainLabel)
        
        emailLabel = UILabel()
        view.addSubview(emailLabel)
    }
```

Reply:
Add the rest of the properties (UI elements) in here. This method should only do that.